packages/dso-toolkit/components/Componenten/banner/_preview.njk

```diff
@@ -17,9 +17,13 @@
   <body>
     {{ yield | safe }}
 
-  <!-- Polyfill for svg use[href] with external uri -->
+    <!-- Polyfill for svg use[href] with external uri -->
```

Comment:
Ik dacht dat ik ergens de hele svg4everybody change eruit had gegooid.. Maar dat zit kennelijk niet in `next`.. Maar wel ff naar kijken.

Reply:
Gedaan in #916